web/dev.ts

```diff
@@ -1405,3 +1406,39 @@ async function importNotificationsFromProd() {
     await importNotifications(prodNotifications.data.notifications, false, true);
     console.log(`Imported notifications from PROD`, prodNotifications.data.notifications);
 }
+
+async function importMessagesTranslationsFromProd() {
+    const prodMessageTranslations = await (
+        await fetch(PROD_URL, {
+            body: JSON.stringify({
+                query: `
+                    query {
+                        notifications {
+                            messageTranslations {
+                                template
+                                id
+                                notificationId
+                                navigateTo
+                            }
+                        }
+                    }
+                `,
+                variables: {},
+            }),
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+            },
+        })
+    ).json();
+
+    let messageTranslations = [];
+
+    prodMessageTranslations.data.notifications.forEach((t) => {
+        if (t.messageTranslations.length > 0) {
+            messageTranslations = [...messageTranslations, ...t.messageTranslations];
+        }
+    });
```

Comment:
I think this is a perfect use case for the reduce function.

```suggestion
    const messageTranslations = prodMessageTranslations.data.notifications.reduce((acc: any[], cur: any) => [...acc, ...cur.messageTranslations], []);
```

Reply:
You're totally right! Thanks!

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat ;)